### PR TITLE
fix typo in QA exclude check, which resulted in all URLs being excluded

### PR DESCRIPTION
**Pull request by ikreymer**
- ensure exclusions now work as expected in replay mode
- add test for using --exclude with replay

### CODE REVIEW

File: tests/qa_compare.test.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ test("run QA comparison, with write pages to redis", async () => {
   fs.rmSync("./test-crawls/qa-wr-net-replay", { recursive: true, force: true });
 
   const child = child_process.exec(
-    "docker run -p 36380:6379 -v $PWD/test-crawls:/crawls webrecorder/browsertrix-crawler qa --qaSource /crawls/collections/qa-wr-net/qa-wr-net.wacz --collection qa-wr-net-replay --crawlId test --qaDebugImageDiff --writePagesToRedis --debugAccessRedis",
+    "docker run -p 36380:6379 -v $PWD/test-crawls:/crawls webrecorder/browsertrix-crawler qa --qaSource /crawls/collections/qa-wr-net/qa-wr-net.wacz --collection qa-wr-net-replay --crawlId test --qaDebugImageDiff --writePagesToRedis --debugAccessRedis --exclude contact",
```
Comment (tw4l) [inline tests/qa_compare.test.js:23]: Maybe could be more specific in the tests below to ensure this was excluded? As is it'll stop checking pages after the third, so the test wouldn't necessarily let us know if contact was included I think.

**Review by tw4l — APPROVED** [review]
Very good catch!

File: tests/qa_compare.test.js (hunk 1 above)
Comment (ikreymer) [reply tests/qa_compare.test.js:23]: done!

Comment (ikreymer) [reply tests/qa_compare.test.js:23]: though check was more to see that we still get 3 pages, if exclude is flipped, we'd exclude anything that wasn't 'contact'

**Pull request merged**